File: src/main/java/com/onysakura/tools/test/TestController.java
package com.onysakura.tools.test;

import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/test")
public class TestController {
    public String hello(String name) {
        return "hello " + name;
    }
}

File: src/main/kotlin/com/onysakura/tools/test/TestController.kt
package com.onysakura.tools.test

import org.springframework.web.bind.annotation.*

@RestController
@CrossOrigin(origins = ["onysakura.com", "https://onysakura.xyz"])
@RequestMapping("/test")
class TestController {

    @GetMapping("/echo")
    fun test(@RequestBody map: MutableMap<String, String>): String {
        println(map)
        return "s"
    }

    @GetMapping("/get")
    fun get() {

    }
}